buổi-9/bai2/bai-2.js

```diff
@@ -0,0 +1,10 @@
+function test() {
+    var age=window.prompt('Nhập tuổi học sinh: ')
+    if(age<16){
+        document.write('Học sinh chưa đủ tuổi vào lớp 10')
+    }
+    else{
```

Comment:
Nhìn kỹ cách viết dấu `}` và `{`
```suggestion
    } else {
```